server/service/src/invoice/inbound_shipment/insert/mod.rs

```diff
@@ -39,6 +41,16 @@ pub fn insert_inbound_shipment(
                 .ok_or(OutError::NewlyCreatedInvoiceDoesNotExist)
         })
         .map_err(|error| error.to_inner_error())?;
+
+    let new_invoice_log = LogRow {
+        id: uuid(),
+        log_type: LogType::InvoiceCreated,
+        user_id: Some(user_id.to_string()),
+        record_id: Some(invoice.invoice_row.id.clone()),
+        created_datetime: Utc::now().naive_utc(),
+    };
+    LogRowRepository::new(&ctx.connection).upsert_one(&new_invoice_log)?;
```

Comment:
Since there will be a lot of these entries I would try to shorten it:

```rust

LogRowRepository::new(&ctx.connection).upsert_one(&LogRow {
        id: uuid(),
        log_type: LogType::InvoiceCreated,
        user_id: Some(user_id.to_string()),
        record_id: Some(invoice.invoice_row.id.clone()),
        created_datetime: Utc::now().naive_utc(),
    })?;
```

maybe even have a log() helper method